src/mongo_client.ts

```diff
@@ -296,6 +325,7 @@ export class MongoClient extends EventEmitter {
       readConcern: this[kOptions].readConcern,
       writeConcern: this[kOptions].writeConcern,
       readPreference: this[kOptions].readPreference,
+      serverApi: this[kOptions].serverApi,
```

Comment:
[nit]: I don't see this being used anywhere, I think the getter on the class will serve to make it easy to access. I only mention it because I thought these `s` objects in general are something we want to get rid of, preferring symbol properties instead.